### ISSUE TITLE
UC4/US6 Collect & present usage information of resources

### ISSUE THREAD

**Issue by pvgenuchten** [soilwise-he/Soilwise-userstory-epics#8]
## User story

As a coordinator of a **soil carbon monitoring network**, 

I want to be able to get an **overview of projects and products** that are enabled by the data and information produced by **my monitoring network**. With open data publications it is often difficult to get a good overview of who is using your data and what products and result are they creating. You often have to rely on mouth to mouth to get this information or follow up scientific publications in detail. Systems requesting information from users to access the data/results such as JRC throw up what can perceived to be an unnecessary barrier, run into GDPR issues and are not able to capture data that is shared further by users. 

So that I can **justify the expenses** and the need for future funding to management and funding authorities.

## Critical success factor: 
This US relies on other US that cover the ingestion of the relevant metadata about projects, datasets and knowledge this US needs to connect to.

## Extended user story description:
I am monitoring network coordinator/data manager, who works at government agency/research institute, which focuses on soil monitoring in Flanders, Belgium in the role of scientist/data manager, my educational level is university (soil scientist, engineer, ….). My main target group are researchers and policy advisors who work in all land use types. I do/don’t have a soil science background. My digital expertise encompasses office programs. I usually use portals google and web of science to find information or data.
I want to be able to connect datasets, projects and knowledge (papers, reports, …) that use data and knowledge produced by my monitoring network to my monitoring network. 
- [ ]	I need to get an overview of scientific papers that use results from my monitoring networks.
- [ ]	I need to get an overview of products and datasets that use results from my monitoring networks.
- [ ]	I want to be able to easily connect from the main entry on my monitoring network to the entries that rely on data and results from my monitoring network.

For example this could be achieved by having a section in the entry with URL’s linking to other sections.
-	Some categorisation could be useful
    -	Data
    -	Report
    -	Paper
    -	….
- I need to be able to determine some statistics form the connections that can be made
    - Statistics:
        - Number of datasets
        - Number of papers
        - Number of reports
             - Type of the reports
                 - Building further on the results
                 - Evaluation of the network
                 - Methodological study
                 - Applied research
                 - Fundamental research
         - Number of and type of Stakeholders
            - Creators of the product
            - Targeted by the product
         - Impact factor of scientific results
         - Open data/open access/FAIR adherence of linkend entries
         - Data quality of the product
         - Themes and domains of the derived products
         - Type and organisations that use the results (scientific, government, business, land managers, citizens, international, …)
         - Nationality
         - Area covered (region of monitoring network, …)
         - …
    -	This can be provided in multiple ways
         - Part or all of these statistics can be calculated in the platform itself and shown as additional information with the entry for the monitoring network
         - A download functionality or API that allows me to pull the necessary metadata to be able to calculate those statistics

## Business reason:
This will allow me to more effectively justify expense and continued investment in my monitoring network to the funding authority. It will better allow me to visualise the impact the network has and have hard figures on the usage. I will be able to reduce the time that needs to be spend to get this information or justify the spending through other means (consultancy, extensive networking, …). ….

## Acceptance criteria:
- [ ] I have an overview of at least four key statics on products using data and results from my monitoring network (numbers of products, impact, data quality fairness, …) or I can pull the necessary data from the repository that allows me to create those key statistics.

**Comment by pvgenuchten** [comment]
meeting 28-2-2024: 
- Linkage between academic articles and used datasets in that research are available via citations on the article. OpenAire offers some linkage between datasets and articles (as added by the article writers). But the article itself may offer additional citations to dataset linkage.
- The frontend should offer some filtering/dashboard mechanism to display results of linked items to a certain dataset
- A user login mechanism is not needed
- Nice to have: Soilwise may derive dataset/knowledge items linkages from context.

**Comment by dluts** [comment]
Extend with information about how results can be shown (API-access is must have and GUI is should have).

Extend with functionality on storing the queries/results (is it needed?) + 

Both for DOI and non-DOI (but with other uuid) content.

User access management functionality, not really necessary at face value, because information is assumed to be public (metadata) or it is assumed that direct access to private information is not necessary, only after processing and aggregation by the SoilWise platform.

Potential new user story on requesting users to add linkages when providing metadata to SoilWise (a new independent functionality from providing metadata or an extension of existing functionality).

**Comment by KathiSchleidt** [comment]
Extracted Requirements:
- 01.1 - Data Discovery - Query Catalogue: Query using a filter to identify metadata records for knowledge and data related to a soil monitoring network. Further filtering between knowledge records and data records, records about resources produced by the network itself and those that are produced by external parties (organization filter, filtering based on the linked references, filtering based on the data sources, ...) and filtering based on target group.
- 01.1 - Data Discovery - Query Catalogue: An overview page for the search results, with access to additional filters (filter options not yet used), for each record a link to the record in soilwise and a short overview (title, preview image, start abstract, type, organization, geographical scope, ...)
- 02 - Metadata creation: Collecting references to current item from other datasets, projects, papers, reports,…
- 02 - Metadata creation: Having an interface or API with analysed references to my datasource
- 02 - Metadata creation: Create metadata about a monitoring network or other project (can be done by the project owners or also automatically if several resources in SoilWise mention a project but no project information is available).
- 02 - Metadata creation: The ability for the user to add linkages between metadata records
- 03 - Metadata feedback: On the metadata record is an overview given of impact statistics of the record
- 04 - Download data: A download functionality or API that allows me to query and pull the enhanced metadata from SoilWise so that the data can be analysed to create statistics about the use of results from a certain project.
- 06 - Data quality checks: Open data/open access/FAIR evaluation of the records
- 08 - Vocabulary: Complex query required vocabs
- 09 - Knowledge: need for metadata on papers, reports, products (scientific and non-scientific), who are related to soil projects and soil monitoring networks. (Not only scientific/soil mission sources).
- 11 - _Processing: analysing the type of references (dataset, report, paper) (out-of-scope)_
- 11 - _Processing: FAIR score (out-of-scope)_
- 11 - _Processing: Link records based on the references mentioned (not only for records with DOI's but for alle records), this should be clickable in UI, queryable (so attributes of the references need to be accesible). Do this linking not only for the references but also based on keywords, information and data mentioned in metadata, tekst and data + keep information about how the link was made and the link type. (out-of-scope)_
- 11 - _Processing: calculating an impact score based on the harvested references, and a large number of other statistics such as number of referecers by type, estimation of the stakeholders,…. (out-of-scope)_
- 13 - Networking: Metadata, core public information about organizations (and maybe also persons?) who are using my data
- 14 - Security: A user can login to edit the autmatically generated record
